### PR TITLE
Issue #13345: Enable example tests for JavadocContentLocation

### PR DESCRIPTION
**Pull request by relentless-pursuit**
part of #13345

### CODE REVIEW

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoccontentlocation/Example1.java [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadoccontentlocation;
+
+class Example1 {
+
 // xdoc section -- start
 /** This comment causes a violation because it starts from the first line
-  * and spans several lines.
-  */
+ * and spans several lines.
+ */
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoccontentlocation/Example1.java:16]: please do me favor and add `private int filed1;` under this comment.

please add some simple fields for all other comments, it took me a while to understand why such comments matters.

**Review by romani — CHANGES_REQUESTED** [review]
items:

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoccontentlocation/Example1.java (hunk 1 above)
Comment (relentless-pursuit) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoccontentlocation/Example1.java:16]: Sure thing.

Comment (relentless-pursuit) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoccontentlocation/Example1.java]: > please do me favor and add `private int filed1;` under this comment.
> 
> please add some simple fields for all other comments, it took me a while to understand why such comments matters.

done

**Review by romani — APPROVED** [review]
Ok to merge

**Comment by relentless-pursuit** [comment]
@romani , can we add hacktoberfest-accepted label?

**Comment by romani** [comment]
@relentless-pursuit , please restore original text as pointed by @rnveach .

File: src/xdocs/checks/javadoc/javadoccontentlocation.xml [hunk 2]
```diff
@@ -112,13 +112,23 @@ public void method();
...
-  */
-/**
-  * This comment is OK because it starts from the second line.
-  */
-/** This comment is OK because it is on the single-line. */
+class Example1 {
+
+  // violation below 'Javadoc content should start from the next line'
+  /** Text
+   * spans several lines.
```
[the remark this answers is not in the document]
Comment (relentless-pursuit) [reply src/xdocs/checks/javadoc/javadoccontentlocation.xml]: done

File: src/xdocs/checks/javadoc/javadoccontentlocation.xml [hunk 3]
```diff
@@ -138,13 +148,24 @@ public void method();
...
-/**
-  * This comment causes a violation because it starts on the second line.
-  */
-/** This single-line comment also is OK. */
+  private int field1;
+
+  // violation below, 'Javadoc content should start from the same line.'
+  /**
+   *
+   * Text.
```
[the remark this answers is not in the document]
Comment (relentless-pursuit) [reply src/xdocs/checks/javadoc/javadoccontentlocation.xml]: done

**Review by rnveach — APPROVED** [review]

**Pull request merged**